version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.0.6-SNAPSHOT"
+version in ThisBuild := "1.0.0"
```

Comment:
We're not ready to be 1.0.0 yet :)

Reply:
if we're introducing breaking changes (like inputs format) - we should bump major version..